Running tests manually on a macos machine
pytest -v tests --ignore=tests/qa --ignore=tests/stress_tests after covalent start is causing most of how to tests to fail after the doc/source/how_to/config/customization.ipynb-0 has run.
Trying to run any covalent CLI command after the tests have run results in the following error since its trying to read the config file
This is presumably happening since the customization-0.ipynb notebook is modifying the config file that results in some entries (enable_logging and log_level) to be written without any section headers.
Snapshot of the config file after the tests have run
To reproduce this issue, manually run the tests via
pytest -v tests --ignore=tests/qa --ignore=tests/stress_tests on MacOS, assert most of the how to tests are failing.
To reproduce the missing header section issue, try to run any covalent cli command i.e. covalent status afterwards
The error traceback should be similar to the following
A potential fix would be to update the customization tutorial, especially those lines that are updating the config with the appropriate section headers i.e
ct.set_config({"enable_logging": "true"}) can be updated to ct.set_config({"sdk.enable_logging": "true"})
Fixed in #797
